README.md

```diff
@@ -6,8 +6,15 @@ A staging instance is currently stood up at `https://vzltaytzg4.execute-api.us-w
 
 ![alt text](diagram.svg "schematic flow representation")
 
-## Using the service
+## Deployment
+We're using the [Chalice microframework](https://chalice.readthedocs.io/en/latest/index.html) to stand up the Lambda. If you want to deploy your own AWS Lambda you need to have an AWS account and your system needs to be configured (see [AWS docs](https://docs.aws.amazon.com/cli/latest/userguide/installing.html) for details). Chalice requires Python 3.6 to be installed on your system. Then follow these steps:
+
+1. Clone [this](https://github.com/DataBiosphere/bagit-firecloud-lambda) repository to your local system and `cd bagit-firecloud-lambda`. 
 
+2. Run `make deploy`. 
+
+This will install two stages of the lambda, `-dev` and one `-staging`. It will deploy a `dev` and a `staging` stage (two separate Lambdas). After you run `make deploy` a subdirectory `.venvbag` has been created in that directory. If you want to delete any of them run for instance `.venvbag/bin/chalice delete --stage dev` to delete the `dev` stage. 
```

Comment:
Should there be a target for this instead? I'm not sure on this one.

Reply:
not sure what you mean by "for this...". target for what?

Reply:
Ah, you mean the delete function? I tried to a teardown that deletes both targets, but it error-ed on deleting the second lambda. perhaps it would work reliable if we separate them into individual targets. It wouldn't make sense in terms of saving number of characters to type: compare `.venvbag/bin/chalice delete --stage dev` to `make teardown-dev`.